src/data/rat/basic.lean

```diff
@@ -304,6 +309,12 @@ begin
   simp only [neg_mul_eq_neg_mul_symm, congr_arg has_neg.neg h₁]
 end
 
+@[simp] lemma mk_neg_denom (n d : ℤ) : n /. -d = -n /. d :=
```

Comment:
How are `-` and `/.` associated? Is this `-(n/.d)` or `(-n)/.d`? I think it might help legibility if you included the parens. If it's the latter, I guess there's another simp lemma that rewrites it to the former? In which case this probably shouldn't be `simp`.

Reply:
In the lemma directly above:
```
@[simp] theorem neg_def {a b : ℤ} : -(a /. b) = -a /. b :=
```
so the answer to your question is that `-` and `/.` associate as `(-n) /. d`. I tried to follow the simp-direction that the `neg_def` lemma defines and simp-normalize to placing the negative inside the fraction, on the numerator. Can you please advise whether that is what you would prefer?

I did not write parens because I understood the mathlib style to not write parens, even when it would be clarifying, if they are already what the association priorities make.

Reply:
Hmm, my guess about what the simp-normal form would be was wrong, but I guess this makes sense. I wouldn't object to unnecessary parens in this case, but I guess they aren't there else where and wouldn't show up in docs, so it's fine. Thanks for clarifying.